src/app/shared/search/search-result.html

```diff
@@ -5,7 +5,8 @@
   <thead>
   <tr>
     <th *ngFor="let col of resultView.columns">
-      <table class="search-result-column-header">
+      <table class="search-result-column-header"
+             attr.aria-label="Sort by {{col.label}} {{ isSortAscending(col)? 'ascending' : 'descending' }}">
```

Comment:
can we find a way to avoid the UI breaking if we forget to add a new type to the search-result-view-item-comparator-factory.ts? 
Could we catch the example for example and log a warning message?

Reply:
Fixed this issue.  Please code review.